src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1241,11 +1747,14 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
   };
 
   private checkMsgSenderIsController = async (txData: Partial<TxData> | undefined) => {
-    assert.assert((await this.controller()) === (await this.getCallerAddress(txData)), 'Msg sender must be controller');
+    assert.assert(
+      (await this.isControllable()) && (await this.controller()) === (await this.getCallerAddress(txData)),
+      'Msg sender must be controller',
+    );
   };
 
   private checkUseModuleVerified = async (address: string) => {
-    if (await (await this.featureRegistryContract()).getFeatureStatus.callAsync(Features.CustomModulesAllowed)) {
+    if (await (await this.featureRegistryContract()).getFeatureStatus.callAsync(Feature.CustomModulesAllowed)) {
```

Comment:
Why is the `ModuleFactoryWrapper` being instanced in `checkModuleCostBelowMaxCost` but the `FeatureRegistryWrapper` isn't being instanced here?

Reply:
Good catch

Reply:
Pushed direct to branch to get code conforming and as little code written as possible